src/Authentication/Controllers/SystemRegisterController.cs

```diff
@@ -107,11 +107,17 @@ public async Task<ActionResult<List<Right>>> GetRightsForRegisteredSystem(string
     /// <param name="cancellationToken">The Cancellationtoken</param>
     /// <returns></returns>
     [HttpPost("system")]    
-    [Authorize(Policy = AuthzConstants.POLICY_SCOPE_SYSTEMREGISTER_WRITE)]
-    public async Task<ActionResult<Guid>> CreateRegisteredSystem([FromBody] RegisterSystemRequest registerNewSystem, CancellationToken cancellationToken = default)
+    ///[Authorize(Policy = AuthzConstants.POLICY_SCOPE_SYSTEMREGISTER_WRITE)]
+    public async Task<ActionResult<Guid>> CreateRegisteredSystem([FromBody] SystemRegisterRequest registerNewSystem, CancellationToken cancellationToken = default)
     {
         try
         {
+            if (!AuthenticationHelper.IsValidOrgIdentifier(registerNewSystem.Vendor))
+            {
+                ModelState.AddModelError("Vendor", "the org number identifier is not valid ISO6523 identifier");
```

Comment:
Give out errors with error codes instead?

Reply:
This error handling will be handled in a new PR. Now we prioritize demo